src/py/flwr/server/app.py

```diff
@@ -367,7 +368,12 @@ def run_superlink() -> None:
 
     if args.isolation == ISOLATION_MODE_SUBPROCESS:
 
-        address = simulationio_address if sim_exec else serverappio_address
+        serverappio_client_address = (
+            SERVERAPPIO_CLIENT_DEFAULT_ADDRESS
+            if serverappio_address == SERVERAPPIO_API_DEFAULT_ADDRESS
+            else serverappio_address
```

Comment:
Instead of this, can we just change the default of `serverappio_address` and then remove this check?

Reply:
@danieljanes I changed my mind: using one IP address `127.0.0.1` throughout as the default makes sense.